templates/default/event/eventinfoshort.html

```diff
@@ -264,7 +264,9 @@
 	{if $event.ticket.state != $smarty.const.RT_RESOLVED}
 	<TR>
 		<TD colspan="3">
-                    {button type="link-button" icon="message" label="New Message" href="?m=rtmessageadd&ticketid={$event.ticketid}"}
+                    {if ($ticket.state != RT_RESOLVED || ($ticket.state == RT_RESOLVED && $aet && ($smarty.now - $ticket.resolvetime) < $aet)) || (ConfigHelper::checkPrivilege('superuser'))}
```

Comment:
Dlaczego nie `$smarty.const.RT_RESOLVED`
Już to dyskutowaliśmy.

Reply:
Uciekło mi poprawiłem.